docs/source/documentation/getting-started.html.md.erb

```diff
@@ -0,0 +1,124 @@
+---
+title: Getting started
+last_reviewed_on: 2025-10-28
+review_in: 6 weeks
+---
+
+# Getting started
+
+This guide will help you set up and run Manage Your Civil Cases locally.
+
+## Prerequisites
+
+- node stable version [24.10.0](https://nodejs.org/en/blog/release/v24.10.0/)
+- [Yarn 4.9.2](https://yarnpkg.com/) package manager (see installation instructions below)
+- TypeScript 5.8.3
+
+### Installing Yarn
+
+This project uses Yarn 4.9.2 managed by corepack (built into Node.js 16.10+). To ensure all team members use the same version, follow these installation steps:
+
+1. **Enable corepack (if not already enabled):**
+
+   ```shell
+   corepack enable
+   ```
+
+2. **Install dependencies:**
+
+   ```shell
+   yarn install
+   ```
+
+3. **Verify the installation:**
+
+   ```shell
+   yarn --version
+   # Should output: 4.9.2
+   ```
+
+**To Note:** 
```

Comment:
Markdown should look like this, to get styles to look right:
```

**To Note:** 

- Corepack automatically uses the Yarn version specified in the `packageManager` field of `package.json`. No additional setup is required once corepack is enabled
- Corepack is the preferred `yarn` way, to install the package manager, instead of `npm install -g yarn` in your ci/cd pipeline
- `yarn install --immutable` ensures that the lockfile (`yarn.lock`) is not modified during the installation process
```